query-tool/src/Piipan.QueryTool/Piipan.QueryTool.csproj

```
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Piipan.QueryTool</RootNamespace>

```

This tweak should enable repeatable package restores per [source tree checklist](https://github.com/18F/piipan/blob/main/docs/source-guidelines.md#source-tree-checklist)

```suggestion
    <RootNamespace>Piipan.QueryTool</RootNamespace>
    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile> and 
    <RestoreLockedMode Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</RestoreLockedMode>
```